src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -38,18 +43,59 @@ public class EventCard extends UiPart<Region> {
     @FXML
     private FlowPane attendees;
 
+    @FXML
+    private ImageView calendarIcon;
+
+    @FXML
+    private ImageView personIcon;
+
     /**
      * Creates a {@code EventCard} with the given {@code Event} and index to display.
      */
     public EventCard(Event event, int displayedIndex) {
         super(FXML);
         this.event = event;
         id.setText(displayedIndex + ". ");
-        name.setText(event.getEventName());
-        date.setText(event.getDate().toString());
-        event.getAttendees().stream()
+        name.setText("  " + event.getEventName());
```

Comment:
We should try to keep the styling purely limited to css, but if it's too complicated to implement, we can refactor this at a later time and just use the whitespace for now

Reply:
Good point, I've made changes in the latest PR to use CSS instead.